presto-docs/src/main/sphinx/release/release-305.rst

```diff
@@ -0,0 +1,53 @@
+===========
+Release 305
+===========
+
+General Changes
+---------------
+
+* Fix failure of :doc:`/functions/regexp` for certain patterns and inputs
+  when using the default ``JONI`` library. (:issue:`350`)
+* Fix a rare ``ClassLoader`` related problem for plugins providing an ``EventListenerFactory``. (:issue:`299`)
+* Un-hide ``join_max_broadcast_table_size`` session property. (:issue:`346`)
+* Improve performance of queries when spill is enabled but not triggered. (:issue:`315`)
+* Consider estimated query peak memory when making cost based decisions. (:issue:`247`)
+* Include revocable memory in total memory stats. (:issue:`273`)
+* Add peak revocable memory to operator stats. (:issue:`273`)
+* Add :func:`ST_Points` function to access vertices of a linestring. (:issue:`316`)
+* Add a system table ``system.metadata.analyze_properties``
+  to list all :doc:`/sql/analyze` properties. (:issue:`376`)
+
+Resource Groups Changes
+-----------------------
+
+* Fix resource group selection when selector uses regular expression variables. (:issue:`373`)
+
+Web UI Changes
+--------------
+
+* Display peak revocable memory, current total memory,
+  and peak total memory, in detailed query view. (:issue:`273`)
+
+CLI Changes
+-----------
+
+* Add option to output CSV without quotes. (:issue:`319`)
+
+Hive Connector Changes
+----------------------
+
+* Fix handling of updated credentials for Google Cloud Storage (GCS). (:issue:`398`)
+* Fix calculation of bucket number for timestamps that contain a non-zero
+  milliseconds value. Previously, data would be written into the wrong bucket,
+  or could be incorrectly skipped on read. (:issue:`366`)
+* Allow writing ORC files compatible with Hive 2.0.0 to 2.2.0 by pretending
```

Comment:
Why is this relevant: "by pretending that the writer is an old version of Hive" ?

Reply:
Because it works by identifying itself as Hive in the file